build/README.md

```diff
@@ -5,6 +5,7 @@ Building Kubernetes is easy if you take advantage of the containerized build env
 ## Requirements
 
 1. Docker, using one of the following configurations:
+  1. **Minikube on Mac/Windows/Linux** you can use [minikube](https://github.com/kubernetes/minikube) to get a Docker build endpoint. Ensure you have at least 3GB of memory. See instructions for [setting up minikube for a build](#minikube).
```

Comment:
word wrap vs one giant line. I know the rest of the doc isn't like this, but let's clean up gradually.

minikube defaults to 2g memory. I know there's a warning here, but we should probably either set the config to use 3, or explicitly talk about the `--memory` flag on `minikube start`.